### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by matthiaskreier**
Fixed link to the first 2 units

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -2,11 +2,11 @@
 
 From functions and state machines to data science and machine learning.
 
-## Unit 1: Functions and Modular Code
+## Unit 1: [Functions and Modular Code](https://github.com/ssis-aa/functions-and-modular-code)
```
Comment (kreier) [inline profile/README.md:6]: Much Better - at least a link

File: profile/README.md [hunk 2]
```diff
@@ -2,11 +2,11 @@
...
 
 In 9 lessons we learn the basics of defining and calling functions, providing input parameters and return output values. And we define and describe the algorithms to solve our problems.
 
-## Unit 2: [Managing State](managing-state)
+## Unit 2: [Managing State](https://github.com/ssis-aa/managing-state)
 
 Convert a binary string into a readable ASCII text to learn more about abstraction and decomposition.
```
Comment (kreier) [inline profile/README.md:11]: This link actually works. Thanks for the fix

**Review by kreier — COMMENTED** [review]
well done. Please fix the next 4 units as well

**Review by matthiaskreier — COMMENTED** [review]
This is fine. Working on it.

**Review by kreier — APPROVED** [review]
Can be merged

**Pull request merged**